### ISSUE TITLE
Using clearer images to replace denoised data in a Multi-Stage Training Process

### ISSUE THREAD

**Issue by hhhhio** [StanfordMIMI/DDM2#36]
Hello,I understand that the first stage involves generating data using a denoising model, which results in data lacking medical details. However, if I have a set of corresponding clearer images for the data used in training, can I replace the first stage generated data with these clearer images? If so, would this be beneficial for the subsequent second and third stages?

Thanks!

**Comment by tiangexiang** [comment]
Yes, clearer images are definitely helpful. However, we need a noise model to be used to estimate the 'state' as in Stage 2. If there is no denoising process in Stage 1 (and no noise model as a result), stage2 is not able to proceed.
